### ISSUE TITLE
Issue with executing rustc command

### ISSUE THREAD

**Issue by sudipta1411** [getcursor/cursor#922]
Whenever I try to execute the `rustc` command, it fails with the following error
 `error: unknown proxy name: 'cursor-0.10.4'; valid proxy names are 'rustc', 'rustdoc', 'cargo', 'rust-lldb', 'rust-gdb', 'rust-gdbgui', 'rls', 'cargo-clippy', 'clippy-driver', 'cargo-miri', 'rust-analyzer', 'rustfmt', 'cargo-fmt'
`

I am using Ubuntu 22.04 with rustc 1.72.0 (5680fa18f 2023-08-23)
I am attaching a screenshot

![Screenshot from 2023-09-13 16-15-06](https://github.com/getcursor/cursor/assets/1386911/cacc2670-00f2-4775-aff5-e7dc949c3322)

**Comment by critesjosh** [comment]
I see this too, with cargo 
```bash
cargo
error: unknown proxy name: 'cursor-0.17.0-build-231127p7iyxn8rg'; valid proxy names are 'rustc', 'rustdoc', 'cargo', 'rust-lldb', 'rust-gdb', 'rust-gdbgui', 'rls', 'cargo-clippy', 'clippy-driver', 'cargo-miri', 'rustfmt', 'cargo-fmt'`
```

**Comment by alanpjohn** [comment]
I am not sure if this issue has been resolved but found a [similar issue in another appimage program](https://github.com/GitSquared/edex-ui/issues/1124) which shows the why this happens. I guess it is only seen by zsh users

**Comment by AlexMikhalev** [comment]
Yes, same issue on Pop!_OS 22.04 LTS.  The issue zsh specific with AppImage. The solution is to switch the terminal inside the cursor to fish or whatever.
